t/test_util.sh

```diff
@@ -4,7 +4,14 @@ wit_repo='git@github.com:sifive/wit'
 test_root=$(dirname $(perl -MCwd -e "print Cwd::realpath('$0')"))
 wit_root=$(perl -MCwd -e "print Cwd::realpath('$test_root/..')")
 
-export PATH=$wit_root:${PATH}
+if [[ -z "${WIT}" ]]; then
```

Comment:
Is this `WIT` environment variable documented anywhere? Also, it's a little unintuitive that a variable named `WIT` should point to the _directory_ containing the `wit` executable, not the `wit` executable itself. Should we consider setting `WIT` to the path of the executable itself, and defaulting to `$wit_root/wit` if it's unset?

Reply:
To be clear, this code doesn't require that `WIT` points to the directory containing `wit`, it also works for the directory containing it, eg.
```
WIT=`which wit` ./t/test_all.sh
```
This will use the result of `which wit`.

Nothing about the tests are documented at all. I'll add a little blurb to the `README-internals`